### PR TITLE
Uhm 4556

### PR DESCRIPTION
**Pull request by mseaton**
Reformatting of Visit dashboard to be tablet-friendly.  Also includes a few changes to:
1) remove "old" encounter templates
2) re-add back in removed ED Triage status in the encounter short template

Screenshots:

**Normal view on Desktop:**

visit-dashboard-desktop

**Mobile views:**

visit-dashboard-mobile-1

visit-dashboard-mobile-2

visit-dashboard-mobile-expanded

### CODE REVIEW

File: omod/src/main/webapp/pages/visit/templates/encounters/checkInEncounterHeading.gsp [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+        <span class="time-span"><i class="icon-time"></i>{{ encounter.encounterDatetime | serverDate:DatetimeFormats.time }}</span>
+    </div>
+</div>
+<div class="row col-12">
+    <span class="encounter-card-summary">{{ encounter.obs | byConcept:Concepts.typeOfVisit:true | obs }}</span>
+</div>
+<div class="row justify-content-between">
+    <div class="col-8">
+        <span class="encounter-card-summary provider-and-location-span">{{ 'emr.by' | translate }} {{ encounter.encounterProviders | encounterRole:primaryEncounterRoleUuid | getProviderName }}</span>
+        <span class="encounter-card-summary admission-location provider-and-location-span" ng-hide="encounter.location.uuid == encounter.visit.location.uuid">${ ui.message("uicommons.at") } {{encounter.location | omrsDisplay}}</span>
```
Comment (brandones) [inline omod/src/main/webapp/pages/visit/templates/encounters/checkInEncounterHeading.gsp:16]: Are there autoformatters for GSP? Does ctrl+alt+L in IntelliJ do anything for this file? These lines are a little bit long

File: omod/src/main/webapp/pages/visit/templates/overview.gsp [hunk 2]
```diff
@@ -1,35 +1,45 @@
...
+                <button id="expand-all-button" ng-click="expandAll()" title="${ ui.message('pihcore.visitNote.expandAll') }"><i class="icon-double-angle-down"/></button>
+            </span>
+
+            <span id="print-actions">
+                <button id="print-button" ng-click="print()" ng-disabled="printButtonDisabled" title="${ ui.message('pihcore.visitNote.print') }"><i class="icon-print"/></button>
+            </span>
+
+        </div>
+
+        <div ng-repeat="encounter in visit.encounters | filter:{voided:false}" class="col-12">
```
Comment (brandones) [inline omod/src/main/webapp/pages/visit/templates/overview.gsp:36]: I think you'll want another div to act as a containing row for this. Right now you've got `row > col > col`

File: omod/src/main/webapp/pages/visit/templates/overview.gsp [hunk 3]
```diff
@@ -1,35 +1,45 @@
...
+            <encounter encounter="encounter" visit="visit" selected="encounter.uuid == encounterUuid" encounter-date-format="encounterDateFormat" country="country" site="site"></encounter>
+        </div>
```
Comment (brandones) [inline omod/src/main/webapp/pages/visit/templates/overview.gsp:38]:
```suggestion
        <div class="row">
          <div ng-repeat="encounter in visit.encounters | filter:{voided:false}" class="col-12">
              <encounter encounter="encounter" visit="visit" selected="encounter.uuid == encounterUuid" encounter-date-format="encounterDateFormat" country="country" site="site"></encounter>
          </div>
        </div>
```

File: omod/src/main/webapp/pages/visit/templates/encounters/checkInEncounterHeading.gsp (hunk 1 above)
Comment (mseaton) [reply omod/src/main/webapp/pages/visit/templates/encounters/checkInEncounterHeading.gsp:16]: Good question.  This is copied directly from "defaultEncounterHeading".  These should really use a single template and pull them in, but I chose not to refactor, just to be consistent.  I'll clean them up.

File: omod/src/main/webapp/pages/visit/templates/overview.gsp (hunk 2 above)
Comment (mseaton) [reply omod/src/main/webapp/pages/visit/templates/overview.gsp:36]: Good catch.

File: omod/src/main/webapp/pages/visit/templates/encounters/vitalsEncounterHeading.gsp [hunk 4]
```diff
@@ -3,17 +3,31 @@
...
+        </span>
     </div>
 </div>
 <div class="row col-12">
-    <span class="encounter-card-summary">{{ encounter.obs | byConcept:Concepts.temperature:true | obs:{ appendComma:true } }} {{ encounter.obs | byConcept:Concepts.heartRate:true | obs:{ appendComma:true } }} {{ encounter.obs | byConcept:Concepts.respiratoryRate:true | obs }}</span>
+    <span class="encounter-card-summary">
+        {{ encounter.obs | byConcept:Concepts.temperature:true | obs:{ appendComma:true } }}
+        {{ encounter.obs | byConcept:Concepts.heartRate:true | obs:{ appendComma:true } }}
+        {{ encounter.obs | byConcept:Concepts.respiratoryRate:true | obs }}
+    </span>
```
Comment (brandones) [inline omod/src/main/webapp/pages/visit/templates/encounters/vitalsEncounterHeading.gsp]: Wow this makes a big difference, thanks!

**Comment by mseaton** [comment]
@mogoodrich and @brandones - I have merged this in, hopefully all looks good.  There will be a followup commit I will need to review and merge in on the smoke tests.

**Pull request merged**